src/frontends/common/include/openvino/frontend/extension/op.hpp

```diff
@@ -25,7 +25,7 @@ inline const ov::OpSet& get_opset_by_name(const std::string& opset_name) {
     if (opsets.find(opset_name) != opsets.end())
         return opsets.at(opset_name)();
     if (opset_name.empty() || opset_name == "latest") {
-        return ov::get_opset14();  // TODO (ticket 138273): Update at the end of the opset15 development
```

Comment:
may be it makes sense to create ticket for opset16 and replace with it?

Reply:
There is no need to mention about Opset16 in the 2024.5 code. 
Initialization of Opset16 is planned to be merged for the next release (2025.0), then such comment can be added.